index.d.ts

```diff
@@ -17,6 +17,9 @@ isAbsoluteUrl('foo/bar');
 //=> false
 ```
 */
-declare function isAbsoluteUrl(url: string): boolean;
+declare function isAbsoluteUrl(
+	url: string,
+	{ httpOnly }?: { httpOnly?: boolean }
```

Comment:
The options should be a separate type and have the same docs as the readme. Should follow: https://github.com/sindresorhus/typescript-definition-style-guide#documentation

Reply:
This is not fixed.